src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -288,6 +291,24 @@ public int SelectedIndex
         /// </value>
         public ObservableCollection<PackageManagerSearchElementViewModel> SearchResults { get; internal set; }
 
+        /// <summary>
+        /// Returns a new filtered collection of packages based on the current user
+        /// </summary>
+        public ObservableCollection<PackageManagerSearchElementViewModel> SearchMyResults
+        {
+            set
+            {
+                if (value == searchMyResults) return;
```

Comment:
The == here only checks for the collection reference and not the collection values. So i suppose this will never be true?

Reply:
What would be good practice here? Should we do the full comparison, or skip the check? This collection is initialized and updated only once per loading the extension, so we wouldn't be assigning new values to it.

Reply:
Yeah we can remove it.

Reply:
Done!